helm/robusta/templates/forwarder.yaml

```diff
@@ -89,4 +89,56 @@ spec:
       {{- if .Values.kubewatch.priorityClassName }}
       priorityClassName: {{ .Values.kubewatch.priorityClassName }}
       {{- end }}
-      
\ No newline at end of file
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ include "robusta.fullname" . }}-forwarder
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app: {{ include "robusta.fullname" . }}-forwarder
+spec:
+  selector:
+    app: {{ include "robusta.fullname" . }}-forwarder
+  ports:
+    - name: forwarder-metrics
+      protocol: TCP
+      port: {{ default 2112 .Values.kubewatch.service.port }}
+      targetPort: {{ default 2112 .Values.kubewatch.service.port }}
```

Comment:
This is the pod port
Why do we need to have the default both here, and in the default values file?
Can it just be in the default values file?

Reply:
Left only default values